scripts/cat/cats.py

```diff
@@ -792,8 +796,9 @@ def gone(self):
         mentors and apprentices."""
         self.outside = True
 
-        if self.status in ["leader", "warrior"]:
-            self.status_change("warrior")
+        if self.status.is_leader() or self.status.is_warrior():
+            # this seems like it should be is_deputy, but I'm doing a like-for-like replacement
```

Comment:
Elaborate, please.

Reply:
The docustring says that this is for handling removing cats from the clan (as in lost). It says it "handles removing them from special positions", and this section of the code is the only part that changes a status. Deputy never gets changed... and I think that changing one's status from warrior to warrior is a little superfluous. Hence my thought that this is supposed to be "is leader or deputy", as in the two special warrior roles.